src/Components/DialPad/dialPad.component.tsx

```diff
@@ -0,0 +1,161 @@
+import React from "react";
+import Holdable from '../holdable/holdable';
+import {
+  Panel,
+  PanelBody,
+  TeamsThemeContext,
+  Input,
+} from "msteams-ui-components-react";
+import {
+  MSTeamsIcon,
+  MSTeamsIconType,
+  MSTeamsIconWeight
+} from "msteams-ui-icons-react";
+import './dialPad.styles.scss';
+import { number } from "prop-types";
+
+interface Props { }
+
+interface State {
+  data: string[],
+  letter: string[],
+  value: string,
+}
+
+
+
+class DialPad extends React.Component<Props, State> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      data: ['1', '2', '3', '4', '5', '6', '7', '8', '9', '*', '0', '#'],
+      letter: ["", "ABC", "DEF", "GHI", "JKL", "MNO", "PQRS", "TUV", "WXYZ", "", "+", ""],
+      value: '',
+    };
+  }
+
+  onHold = (evt: any) => {
```

Comment:
should depend on the selected button:
for example, a long click on the number (0) will generate a (+) , not all numbers.

Reply:
ok , I will do .